src/formula/callable_objects.hpp

```diff
@@ -25,10 +25,34 @@ class team;
 class terrain_type;
 class unit;
 class unit_type;
+namespace game_events {struct queued_event;}
 
 namespace wfl
 {
 
+// An object representing the state of the game, providing access to the map and basic information
+class gamestate_callable : public formula_callable
+{
+public:
+	// TODO: Constructor
```

Comment:
This TODO should be resolved before merging.

Reply:
Is simply deleting it sufficient? I put it there thinking there must be some object I need to pass to it to encompass the game state, but then I ended up reading everything from the `resources` namespace instead.

Reply:
That'd be fine.